src/index.html

```diff
@@ -9,10 +9,43 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute is pointing to a SCSS file (`styles/main.scss`). Browsers cannot directly interpret SCSS files; they need to be compiled into CSS first. Ensure that you compile your SCSS into a CSS file and link to that CSS file instead.